extensions/transmission/src/modules/client.ts

```diff
@@ -127,3 +127,7 @@ export const useSessionStats = ({ refreshInterval = INTERVAL }: { refreshInterva
     { refreshInterval }
   );
 };
+
+export const isLocalTransmission = () => {
+  return preferences.host === "localhost" || preferences.host === "127.0.0.1";
```

Comment:
I think some users reported localhost wasn't working and they were required to use 127.0.0.1 (or vice versa?), it may make sense to enforce the correct format maybe. Not blocking for this PR of course.

Reply:
Yeah that might be good, I think I had a similar issue. This particular function was just defensive coding to make sure that any expectedly local value will work, not necessarily sufficient validation on the input or anything like that.

Reply:
Should we wait for a push, or just continue with the merge?

Reply:
I would address that in a separate pr